tools/appveyor.psm1

```diff
@@ -108,6 +108,20 @@ function Invoke-AppVeyorInstall
         Update-AppveyorBuild -message $buildName
     }
 
+    # Generate credentials for appveyor remoting tests.
```

Comment:
You are saying you can't run enable-rdp.ps1 again?
@TravisEz13 Can you make this general for AppVeyor?  I am not very familiar with AppVeyor and would prefer that you made the correct changes.